app/src/main/java/com/arygm/quickfix/ui/search/SearchWorkerProfileResult.kt

```diff
@@ -0,0 +1,150 @@
+                      fontFamily = poppinsFontFamily,
+                      color = colorScheme.onBackground)
+
+                  Text(text = name, fontSize = 9.sp, fontWeight = FontWeight.Bold)
+                }
+                Row(verticalAlignment = Alignment.Bottom) {
+                  Text(
+                      text = "CHF",
+                      fontSize = 13.sp,
+                      fontWeight = FontWeight.SemiBold,
+                      lineHeight = 20.sp,
+                      color = colorScheme.onBackground)
+                  Text(
+                      text = price,
+                      fontSize = 19.sp,
+                      fontWeight = FontWeight.Bold,
+                      lineHeight = 20.sp,
+                      color = colorScheme.onBackground)
+                  Text(
+                      text = "/Hour",
+                      fontSize = 13.sp,
+                      fontWeight = FontWeight.SemiBold,
+                      lineHeight = 20.sp,
+                      color = colorScheme.onSurface)
+                }
+              }
+
+              Column(
+                  horizontalAlignment = Alignment.End,
+                  verticalArrangement = Arrangement.SpaceBetween,
+                  modifier = Modifier.weight(0.4f).height(100.dp).padding(end = 8.dp)) {
+                    Row(verticalAlignment = Alignment.CenterVertically) {
+                      Icon(
+                          imageVector = Icons.Default.LocationOn,
+                          contentDescription = "Location",
+                          tint = colorScheme.onSurface,
+                          modifier = Modifier.size(16.dp))
+                      Text(text = location, fontSize = 11.sp, color = Color.Gray)
+                    }
+                    Row(verticalAlignment = Alignment.Bottom) {
+                      QuickFixButton(
+                          onClickAction = onBookClick,
+                          buttonText = "Book",
+                          textStyle =
+                              poppinsTypography.bodyMedium.copy(
+                                  fontWeight = FontWeight.SemiBold,
+                                  fontSize = 16.sp,
+                                  lineHeight = 20.sp),
+                          buttonColor = colorScheme.primary,
+                          textColor = colorScheme.onPrimary,
+                          contentPadding = PaddingValues(0.dp),
+                          height = 30.dp)
+                    }
+                  }
+            }
+      }
+}
+
+@Preview(showBackground = true)
+@Composable
+fun PreviewSearchWorkerProfileResult() {
+  SearchWorkerProfileResult(
+      profileImage = R.drawable.placeholder_worker,
+      name = "Moha Abbes",
+      category = "Exterior Painter",
+      rating = 4.0f,
+      reviewCount = 160,
+      location = "Rennens",
+      price = "42",
+      onBookClick = { /* Handle book click in preview */})
+}
```

Comment:
I don't think this should be kept on the main branch, even though it's a nice way of visualizing the result.